packages/manager/modules/pci/src/projects/project/ai-dashboard/style.scss

```diff
@@ -0,0 +1,63 @@
+ovh-manager-pci-project-a-i-dashboard {
+  .oui-box-block {
+    &__heading {
+      border: solid 1px #c4ecfe;
+      background-color: #f5fefe;
+      font-size: 1.2rem;
+      padding: 0.4rem;
+      color: #3257d8;
+      font-weight: 700;
+    }
+    &__body {
+      padding: 1rem;
+      border: solid 1px #c4ecfe;
+
+      &__content {
+        border-bottom: solid 1px #f2f3f2;
+        margin-bottom: 1rem;
+        padding-bottom: 1rem;
+      }
+    }
+  }
+
+  .squared-box {
+    display: flex;
+    flex-direction: column;
+    padding: 1rem 3rem 1rem 3rem;
+    align-items: center;
+
+    &__value {
+      line-height: 1;
+      font-size: 5rem;
+      font-weight: 700;
+    }
+    &__description {
+      font-weight: 700;
+      color: #3257d8;
+    }
+  }
+
+  .colored-gradient {
+    background: linear-gradient(105deg, #e6f464, #43feed);
+  }
+
+  .icon-badge {
+    font-size: 2rem;
+    background: rgb(239, 249, 254);
+    padding: 1rem;
+    width: 4rem;
+    height: 4rem;
+    border-radius: 100%;
+    margin-right: 1rem;
+  }
+
+  .billing-price {
+    font-size: 3rem;
+    color: #268302;
+    font-weight: 700;
+  }
+
+  a {
+    vertical-align: middle;
+  }
```

Comment:
**nitpick:** Avoid custom CSS and please re-use the utilities available in bootstrap and ui-kit

Reply:
This has been discussed prior with both CX and Core team, we need custom css here, because new components are not ready yet. We have been authorized to write our own for this project, as long as the styles are scoped to the module.